Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionPrivate.h

```diff
@@ -96,6 +96,8 @@ WK_HEADER_AUDIT_BEGIN(nullability, sendability)
 /*! @abstract A Boolean value indicating whether the extension has a sidebar. */
 @property (readonly, nonatomic) BOOL _hasSidebar;
 
+@property (readonly, nonatomic, copy) NSURL *_resourceURL;
```

Comment:
Ditto: https://github.com/WebKit/WebKit/pull/37729/files#r1879148684

Reply:
yes, but this PR wen up before that discussion. just updated the PR